modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/blockstream/BlockStreamContext.kt

```diff
@@ -0,0 +1,15 @@
+package jp.co.soramitsu.iroha2.client.blockstream
+
+import io.ktor.client.HttpClient
+import jp.co.soramitsu.iroha2.generated.VersionedBlockMessage
+import java.net.URL
+
+data class BlockStreamContext(
+    val apiUrl: URL,
+    val client: HttpClient,
+    val from: Long = 1,
+    val count: Int? = null,
```

Comment:
rename to smth like `receivedBlockCountCap`

Reply:
I think we have a comment to describe the purpose of the parameter
I'd leave it as it is because short variable names are always better)

Reply:
or if you can suggest some other short name we can discuss it

Reply:
suggesting to remove this at all